static/js/components/ChannelHeader_test.js

```diff
@@ -23,27 +23,29 @@ describe("ChannelHeader", () => {
         channel={channel}
         isModerator={false}
         history={history}
-        hasNavbar={true}
         {...props}
       />
     )
 
-  it("renders a channel page header", () => {
+  it("renders a channel header", () => {
     const wrapper = render()
     assert.deepEqual(
       wrapper.find("Connect(ChannelAvatar)").prop("channel"),
       channel
     )
-
-    const links = wrapper.find("IntraPageNav NavLink")
-    assert.equal(links.length, 1)
-    const props = links.props()
-    assert.equal(props.to, channelURL(channel.name))
-
     const linkProps = wrapper.find("Link").props()
     assert.equal(linkProps.to, channelURL(channel.name))
     assert.equal(linkProps.children, channel.title)
   })
+  ;[true, false].forEach(hasNavbar => {
+    it(`${hasNavbar ? "has" : "doesn't have"} navbar items`, () => {
```

Comment:
Minor nitpick, feel free to ignore - you can use `shouldIf` here:
```javascript
it(`${shouldIf(hasNavbar)} have navbar items`
```

Reply:
PS conflict needs to be resolved too

Reply:
Sounds good